db/schema.rb

```diff
@@ -48,9 +48,9 @@
     t.datetime "pledge_generated_at", precision: nil
```

Comment:
oh you have found our [weird integer -> bigint bug](https://github.com/DARIAEngineering/dcaf_case_management/pull/2571)! we're not sure what's triggering this but we definitely don't want to change these in this PR! Can you drop those schema changes?

Reply:
Oh whoops...reverted now!